.github/workflows/ci.yml

```diff
@@ -110,8 +110,8 @@ jobs:
       - name: Fetch Boost.CI
         uses: actions/checkout@v2
         with:
-          repository: boostorg/boost-ci
-          ref: master
+          repository: Flamefire/boost-ci
+          ref: feature/codecov-setup
```

Comment:
Is that right?

Reply:
Just for testing. Sorry should have marked this as a draft

Reply:
No problem, just wasn't sure.

Reply:
@mloskot I added a commit to skip the copy of the ci folder if running for the boost-ci repo. This way the PR tests the new files without the need for such a change